Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.cpp

```diff
@@ -1135,8 +1186,6 @@ namespace AZ::SettingsRegistryMergeUtils
         AZStd::array commandOptions = {
             OptionKeyToRegsetKey{
                 "engine-path", AZStd::string::format("%s/engine_path", BootstrapSettingsRootKey)},
-            OptionKeyToRegsetKey{
-                "project-path", AZStd::string::format("%s/project_path", BootstrapSettingsRootKey)},
             OptionKeyToRegsetKey{
```

Comment:
It might be helpful to update the comment on line 1185 that still mentions the project root override

Reply:
Will do.